src/app/components/elements/inputs/DateInput.tsx

```diff
@@ -143,7 +143,7 @@ export const DateInput = (props: DateInputProps) => {
   function calculateHiddenValue(): string {
     const timestamp = calculateDate();
     if (timestamp) {
-      return timestamp.toISOString().substr(0, 10);
+      return timestamp.toISOString().substring(0, 10);
```

Comment:
You may already be aware of this, but just in case: this only works because the first parameter is 0. If it had been anything else, then this wouldn't work, as the second parameter means something different in those two functions. I'm only saying this in case you need to replace the deprecated `substr` again, be careful because it's not interchangeable with `substring`, they do different things.

Reply:
Yeah I did initially put 11 but then realised 0,10 worked :) I double checked that the value was what I expected

Reply:
Yeah I did initially put 11 but then realised 0.10 worked :) I double checked that the value was what I expected to be passed over too